modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/labels/DateLabelProcessor.java

```diff
@@ -46,6 +46,12 @@ public class DateLabelProcessor extends SimpleAnswerLabelProcessor implements Re
 {
     private static final DateFormat DEFAULT_FORMAT = SimpleDateFormat.getDateInstance();
 
+    @Override
+    public String getDescription()
+    {
+        return super.getDescription() + "Get the human readable answer for date questions.";
```

Comment:
To be reformatted because the updated line is too long

```suggestion
        return super.getDescription() + "Get the human readable answer for date questions. The human readable version is the date formatted with the date format configured in the date question definition, for example `01/07/2025` instead of the stored value `2025-01-07T00:00:00%5C.000-05:00`.";
```